rosidl_generator_py/resource/_msg_support.entry_point.c.template

```diff
@@ -1,85 +1,136 @@
 
-      PyObject *pyconvert_to_py = NULL;
-      pyconvert_to_py = PyCapsule_New((void *)&@(spec.base_type.pkg_name)_@(module_name)__convert_to_py, NULL, NULL);
-      if (pyconvert_to_py != NULL) {
-        if (PyModule_AddObject(m, "convert_to_py", pyconvert_to_py)) {
-          Py_XDECREF(pyconvert_from_py);
-          pyconvert_from_py = NULL;
-          Py_XDECREF(pyconvert_to_py);
-          pyconvert_to_py = NULL;
-          Py_XDECREF(m);
-          m = NULL;
-        }
-        PyObject *pytype_support = NULL;
-        pytype_support = PyCapsule_New(
-          (void *)ROSIDL_GET_MSG_TYPE_SUPPORT(@(spec.base_type.pkg_name), @(spec.msg_name)),
-          NULL, NULL);
-        if (pytype_support == NULL) {
-          Py_XDECREF(pyconvert_from_py);
-          pyconvert_from_py = NULL;
-          Py_XDECREF(pyconvert_to_py);
-          pyconvert_to_py = NULL;
-          Py_XDECREF(m);
-          m = NULL;
-        } else {
-          if (PyModule_AddObject(m, "type_support", pytype_support)) {
-            Py_XDECREF(pytype_support);
-            pytype_support = NULL;
-            Py_XDECREF(pyconvert_from_py);
-            pyconvert_from_py = NULL;
-            Py_XDECREF(pyconvert_to_py);
-            pyconvert_to_py = NULL;
-            Py_XDECREF(m);
-            m = NULL;
-          }
-        }
-      }
-    } else {
+    pyconvert_from_py_@(module_name) = PyCapsule_New(
+                                          (void *)&@(spec.base_type.pkg_name)_@(module_name)__convert_from_py,
+                                          NULL, NULL);
+    if (pyconvert_from_py_@(module_name) == NULL) {
+      DECREF_@(lastobject.upper())
       Py_XDECREF(m);
       m = NULL;
+      return m;
     }
-  }
-  return m;
+@{
+lastobject = 'pyconvert_from_py_' + module_name    
+}@
+    if (PyModule_AddObject(m, "convert_from_py_@(module_name)", pyconvert_from_py_@(module_name))) {
+      DECREF_@(lastobject.upper())
+      Py_XDECREF(m);
+      m = NULL;
+      return m;
+    }
+
+    PyObject *pyconvert_to_py_@(module_name) = NULL;
+    pyconvert_to_py_@(module_name) = PyCapsule_New(
+                                        (void *)&@(spec.base_type.pkg_name)_@(module_name)__convert_to_py,
+                                        NULL, NULL);
+    if (pyconvert_to_py_@(module_name) == NULL) {
+      DECREF_@(lastobject.upper())
+      Py_XDECREF(m);
+      m = NULL;
+      return m;
+    }
+@{
+lastobject = 'pyconvert_to_py_' + module_name    
+}@
+    if (PyModule_AddObject(m, "convert_to_py_@(module_name)", pyconvert_to_py_@(module_name))) {
+      DECREF_@(lastobject.upper())
+      Py_XDECREF(m);
+      m = NULL;
+      return m;
+    }
+    PyObject *pytype_support_@(module_name) = NULL;
+    pytype_support_@(module_name) = PyCapsule_New(
+                                      (void *)ROSIDL_GET_TYPE_SUPPORT(@(spec.base_type.pkg_name), @(subfolder), @(spec.msg_name)),
+                                    NULL, NULL);
```

Comment:
same here, also inconsistent. You might want to run our linter over one of the generated files just to see what issues come up. You don't have to have the generated files address all the linter issues (line length is particularly hard to guarantee), but I would appreciate it if you can fix some of the other style issues the linter reports. It might also help you catch non-style issues if you run `cppcheck` over it, even if it's only once.